docs/intro.rst

```diff
@@ -108,7 +108,7 @@ Pywbem is supported in these environments:
 Limitations:
 
 * On Windows (native), pywbem is not supported on Python 2.6, because the
-  M2CryptoWin32/64 packages do not support Python 2.6.
+  M2Crypto package does not support Python 2.6.
```

Comment:
Do you mean the M2Crypto windows package???

Reply:
This PR changes from using the outdated M2CryptoWin32/64 packages to using the M2Crypto package, on Windows.

Reply:
So should say that M2Crypto package on Windows does not support ...

Reply:
That's what it says, if you read the whole sentence.